CHANGES.md

```diff
@@ -15,6 +15,16 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
+## Fixes
+
+- Wrong compose up extra arguments given to compose restart
+
+  * When setting `BIRDHOUSE_COMPOSE_UP_EXTRA_OPTS="--remove-orphans"` in
```

Comment:
Can you reword this so that it's clear that `--remove-orphans` is one example but that this applies to any additional option that only applies to `up`.

Reply:
done https://github.com/bird-house/birdhouse-deploy/pull/515/commits/3a2732fb72759a87e8667ed7fac79c9004438189